addon/components/lazy-image.js

```diff
@@ -6,17 +6,20 @@ var get       = Ember.get;
 var set       = Ember.set;
 var observer  = Ember.observer;
 var Component = Ember.Component;
+var defaultLazyImage = "assets/lz-image/lazy_1x1.gif";
```

Comment:
to avoid using assets, I added [this](https://github.com/twokul/ember-lazy-image/blob/master/app/templates/components/lazy-image.hbs#L4). It works fine for modern browsers. If there's a need to support IE, you can override with your own placeholder.